app/services/no_bp_measure_service.rb

```diff
@@ -38,32 +38,32 @@ def cache_keys_for_period
 
   def execute_sql(period)
     return 0 if facility_ids.blank?
-    start_date = period.blood_pressure_control_range.begin
-    end_date = period.blood_pressure_control_range.end
-    registration_date = period.blood_pressure_control_range.begin
+    start_time = period.blood_pressure_control_range.begin
+    end_time = period.blood_pressure_control_range.end
+    registration_buffer = period.blood_pressure_control_range.begin
```

Comment:
Should this be using `adjusted_period` instead? This is confusing and I suppose it works because `blood_pressure_control_range.begin` and `adjusted_period.end` are the same. Should use the right method here and in `NoBPMeasureQuery `.